tests/test_rpmutils.py

```diff
@@ -59,8 +59,8 @@ def test_no_component(self):
                 ('', '1.2', '', '', ''))
 
     def test_absurd(self):
-        assert (split_filename('1:aarch64-ppc64le-i686-0.1-1.el7ost.src.rpm') ==
-                ('aarch64-ppc64le-i686', '0.1', '1.el7ost', '1', ''))
+        assert (split_filename('1:aarch64-ppc64le-i686-0.1-1.el7ost.src.rpm') == # NOQA
+                ('aarch64-ppc64le-i686', '0.1', '1.el7ost', '1', '')) # NOQA
```

Comment:
Rather than ignoring these, the problem could be solved if the values were extracted to variables